templates/web/pages/tag/tag.tt.html

```diff
@@ -48,6 +50,42 @@
             [% description %]
         </div>
     [% END %]
+    [% IF tagid.defined  %]
+        <!-- injecting facet-knowledge-panel -->
+        <div id="facet-knowledge-panel">
+            <h2 id="title"></h2>
```

Comment:
Please prefix all ids with facet_panels_ or something, "title" is too generic.

Reply:
Okay!